builds.go

```diff
@@ -0,0 +1,359 @@
+//
+// Copyright 2016, Arkbriar
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package gitlab
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/url"
+	"time"
+)
+
+// ListBuildsOptions are options for two list apis
+type ListBuildsOptions struct {
+	Scope []string `url:"scope,omitempty", json:"scope,omitempty"`
+}
+
+// BuildsService handles communication with the ci builds related methods
+// of the GitLab API.
+//
+// GitLab API docs: http://docs.gitlab.com/ce/api/builds.html
+type BuildsService struct {
+	client *Client
+}
+
+// Runner represents a ci runner
+//
+// GitLab API docs: http://docs.gitlab.com/ce/api/builds.html
+type Runner struct {
+	ID          int    `json:"id"`
+	Description string `json:"description"`
+	Active      bool   `json:"active"`
+	IsShared    bool   `json:"is_shared"`
+	Name        string `json:"name"`
+}
+
+// ArtifactsFile represents an artifacts file (.gitlab.yml)
+//
+// GitLab API docs: http://docs.gitlab.com/ce/api/builds.html
+type ArtifactsFile struct {
+	Filename string `json:"filename"`
+	Size     int    `json:"size"`
+}
+
+// Build represents a ci build.
+//
+// GitLab API docs: http://docs.gitlab.com/ce/api/builds.html
+type Build struct {
+	Commit        Commit        `json:"commit"`
```

Comment:
Please use a pointer here...

Let me explain a little why this should have a pointer and the previous ones don't. It is not possible to pass around an anonymous struct to another function or variable, so there is no use in it being a pointer. But defined types like `Commit` and `User` can be passed around. So they being a pointer, does make sense :wink: